propagators/b3/b3_propagator_test.go

```diff
@@ -217,6 +217,16 @@ func TestExtractSingle(t *testing.T) {
 		{"000000000000007b", trace.SpanContextConfig{}, errInvalidScope, false, false},
 		{"000000000000007b00000000000001c8", trace.SpanContextConfig{}, errInvalidScope, false, false},
 		// Support short trace IDs.
```

Comment:
This comment seems to apply to the other test cases, not the two you just added.
There should probably be a quick explanation for those two cases, such as `Handle invalid Trace and Span IDs`.

Reply:
Sorry, my bad. I have resolved them.